style.css

```diff
@@ -0,0 +1,553 @@
+  font-size: 2rem;
+  padding: 1rem;
+  transition: all 0.2s ease-out;
+}
+
+#menu-btn:hover {
+  color: var(--orange);
+  transform: scale(1.2);
+}
+
+/* || Mobile Menu  */
+.mobile-menu {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+  position: absolute;
+  width: 100%;
+  height: 100vh;
+  top: 0;
+  left: 0;
+  background-color: #272a31c4;
+  backdrop-filter: blur(5px);
+  overflow-y: scroll;
+  z-index: 5;
+}
+
+.mobile-menu .nav-menu {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+  margin-bottom: 1rem;
+}
+
+.mobile-menu .nav-menu a {
+  padding: 0.5rem 1rem;
+  border: none;
+  width: 100%;
+  font-size: 1.8rem;
+  font-weight: 700;
+  color: #fff;
+}
+
+.mobile-menu .active a {
+  color: var(--orange);
+}
+
+.mobile-menu .nav-menu a:hover {
+  background-color: var(--orange);
+  color: #fff;
+}
+
+.mobile-menu .nav-btn {
+  display: none;
+}
+
+.mobile-toolbar {
+  display: none;
+  border-top: solid 1px;
+  margin: 1rem 0;
+  width: 100%;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: repeat(2, 1fr);
+}
+
+.mobile-toolbar li {
+  padding: 1rem 0.5rem;
+  text-align: center;
+}
+
+.mobile-tool-link {
+  display: block;
+  font-size: 1.5rem;
+  transition: all 0.3s ease;
+  color: #fff;
+}
+
+.mobile-tool-link:hover {
+  color: var(--orange);
+  transform: scale(1.2);
+}
+
+.mobile-toolbar i {
+  font-size: 2rem;
+}
+
+.mobile-menu .mobile-toolbar {
+  display: grid;
+}
+
+.mobile-menu #menu-btn {
+  color: #fff;
+  transition: all 0.3s ease;
+}
+
+.mobile-menu #menu-btn:hover {
+  transform: rotateZ(90deg);
+  color: var(--orange);
+}
+
+/* || Headline Section  */
+.headline-container {
+  background-color: var(--gray);
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+  padding: 2rem 1rem;
+}
+
+.headline h1 {
+  /* color: var(--orange); */
+  font-size: 3rem;
+  font-weight: 900;
+  color: transparent;
+  background-image: linear-gradient(45deg, #ff1900, #e7b930);
+  background-clip: text;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
+
+.greeting {
+  display: block;
+  font-weight: 700;
+  font-size: 1.6rem;
+}
+
+.event-date {
+  font-size: 1.8rem;
+  font-weight: 900;
+}
+
+.event-location {
+  font-size: 1.3rem;
+}
+
+/* || Program Section  */
+.program-container {
+  position: relative;
+  background-image: url('img/chessbackground.jpg');
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  color: #fff;
+}
+
+.program-container .section-heading {
+  z-index: 5;
+}
+
+.program-container hr {
+  z-index: 5;
+}
+
+.overlay-bg {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: #16171bce;
+}
+
+.program-card {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 0.25rem;
+  padding: 1rem;
+  background-color: rgba(51, 51, 51, 0.879);
+  z-index: 5;
+}
```

Comment:
- [ ] [REQUIRED] Good job so far 👍 Please add `hover` effect for cards like [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏 

### Your design ❌ 

![image](https://user-images.githubusercontent.com/63915024/203613058-b7a20b41-842f-41f6-bb23-7d2dfe376219.png)

### [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) ✔️ 

![image](https://user-images.githubusercontent.com/63915024/203613014-e4e0c2f7-b09d-4e20-ae5c-48a2fed72a0c.png)